core/src/main/java/nl/nn/adapterframework/extensions/javascript/Nashorn.java

```diff
@@ -54,7 +54,11 @@ public void startRuntime() {
 		if (StringUtils.isNotEmpty(alias))
 			executeScript(alias + " = this;");
 
-		executeScript("load('classpath:net/arnx/nashorn/lib/promise.js')");
+		try {
+			engine.eval("load('classpath:net/arnx/nashorn/lib/promise.js')");
+		} catch (NullPointerException | ScriptException e) {
```

Comment:
Ik zou in plaats van `catch (NullPointerException | ScriptException e)` schrijven: `catch (Exception e)`, ook bij de verderop volgende methods